src/dashboard/main.py

```diff
@@ -49,7 +49,7 @@ def page_container() -> Component:
     dash.page_container.className = "grow overflow-auto"
     return html.Div(
         id="main",
-        className="flex h-screen overflow-x-hidden",
+        className="flex bg-[#E9E9F2] h-screen overflow-x-hidden",
```

Comment:
I'm not sure this should be applied to the page container rather than the page itself?
Consider using `bg-background`.